### PR TITLE
[feature] implement transfer limits for accounts not on chain 

### PR DESCRIPTION
**Pull request by 0xzoz**
- This adds the feature mentioned in #184 which adds a limit to the amount that can be sent to accounts that do not currently exist on chain

### CODE REVIEW

File: framework/libra-framework/sources/ol_sources/ol_account.move [hunk 1]
```diff
@@ -215,8 +226,12 @@ module ol_framework::ol_account {
         coin
     }
 
-    fun maybe_sender_creates_account(sender: &signer, maybe_new_user: address) {
+    fun maybe_sender_creates_account(sender: &signer, maybe_new_user: address,
+    amount: u64) {
       if (!account::exists_at(maybe_new_user)) {
+          // prevents someone's Terrible, Horrible, No Good, Very Bad Day
```
Comment (0xslipk) [inline framework/libra-framework/sources/ol_sources/ol_account.move:232]: ❤️

File: framework/libra-framework/sources/ol_sources/tests/account.test.move [hunk 2]
```diff
@@ -8,7 +8,8 @@ module ol_framework::test_account {
   use ol_framework::ol_account;
   use ol_framework::ancestry;
   use diem_framework::coin;
-  use diem_std::debug::print;
+  // use diem_std::debug::print;
```
Comment (dboreham) [inline framework/libra-framework/sources/ol_sources/tests/account.test.move:11]: Could this commented line be removed?

File: framework/libra-framework/sources/ol_sources/ol_account.move [hunk 3]
```diff
@@ -363,13 +378,20 @@ module ol_framework::ol_account {
...
-        // multply will TRUNCATE.
+        // multiply will TRUNCATE.
         let integer_part = fixed_point32::multiply_u64(1, value);
 
         let decimal_part = unscaled_value - (integer_part * scaling);
 
         (integer_part, decimal_part)
     }
+
+    fun scale_from_human(human: u64): u64 {
```
Comment (dboreham) [inline framework/libra-framework/sources/ol_sources/ol_account.move:389]: Checking this function is included intentionally because afaics it isn't called.

**Review by dboreham — APPROVED** [review]

File: framework/libra-framework/sources/ol_sources/tests/account.test.move (hunk 2 above)
Comment (0o-de-lally) [reply framework/libra-framework/sources/ol_sources/tests/account.test.move:11]: ugly, but I keep these in there for convenience when I need to print stuff in debug.

**Pull request merged**